### ISSUE TITLE
The text-panel disappears by clicking the "zoom" tile

### ISSUE THREAD

**Issue by highfield** [vvvmax/unitegallery#49]
"Grid" theme, full of thumbnails.
When I click on any thumbnail, the related full image is displayed correctly, as well as the text-panel (description) on the bottom part of the viewport.
The problem arises when I press the "zoom" tile: the picture is correctly scaled, but the text-panel fades out. I may try to click any button, but unless selecting another thumbnail, there's no way to take the text-panel visible again.

**Comment by vvvmax** [comment]
HI, please write me a support ticket here:
valiano.ticksy.com

please specify the website url where you have this bug, I'll check.

**Comment by highfield** [comment]
I'd create the ticket, but the available categories don't show the "plain" library. Let me know how to specify it, and I'll do sooner.
BTW, the problem is easy to see even on your web page:
http://unitegallery.net/index.php?page=grid-right
The first picture you see is "Rocks on water". Just press the "zoom in" (+) button and the bottom strip with the photo caption disappears. The only way to see it again is by selecting another picture from the thumbs grid, or swiping back and forth.

**Comment by vvvmax** [comment]
Yes, I made it in purpose. if you want to see the image, then the caption
is distructing you. so when the image zooms in, the caption dissapears.

**Comment by highfield** [comment]
Okay, but is there any way to re-display the caption without be forced to change picture?
When I restore the normal size (the 3rd button in the top-left of the frame), why don't I have my caption back?

**Comment by vvvmax** [comment]
it's not coming back because I assume that if you zoom you don't need the
caption anymore :)
it's built like this, sorry.
